ui/k-info-panel/show_trace_results.kytos

```diff
@@ -0,0 +1,119 @@
+<template>
+  <k-accordion>
+    <k-accordion-item title="Trace Results">
+      <k-context-panel v-if="is_empty" title="Empty Result: Verify Inputs" icon="gear">
```

Comment:
Maybe we can just leave "Empty Result" and not necessarily "Verify Inputs"? Check out my comment below regarding that a valid trace indeed could just result in an empty result (for instance when a trace doesn't match in the first switch)